interfaces/builtin/steam_support.go

```diff
@@ -0,0 +1,201 @@
+
+# i.e. /dev/shm/mono.16965
+owner /{dev,run}/shm/mono.@{pid} mrw,
+
+
+########################
+#     GPU Access       #
+########################
+
+# Games can and will attempt to map /dev/nvidia* (specifically Unity)
+/dev/nvidiactl mr,
+/dev/nvidia[0-9] mr,
+
+########################
+#    Vulkan Support    #
+########################
+
+# Base runtime can contain the vulkan loader itself
+/usr/share/vulkan/explicit_layer.d/ r,
+/usr/share/vulkan/explicit_layer.d/** r,
+
+# Base runtime can contain mesalib definitions for Vulkan ICD files
+/usr/share/vulkan/icd.d/ r,
+/usr/share/vulkan/icd.d/** r,
+
+########################
+#      Disks/Media     #
+########################
+
+# Steam requires +x permissions on the partitions and will perform
+# such a test:
+# sh: /run/media/bigdisk/games//steamapps/.steam_exec_test.sh: /bin/sh: bad interpreter: Permission denied
+/{,run/}media/**/.steam_exec_test.sh ixmrw,
+
+# Libraries and executables on other partitions require map + execute permissions
+/{,run/}media/**/steamapps/common/** ixm,
+
+########################
+# Udev / PCI Support   #
+########################
+
+# Steam Controller
+/run/udev/data/c247:[0-9]* r,
+/run/udev/data/c248:[0-9]* r,
+
+# Allow thread-synchronisation to work in the Steam client
+/sys/devices/pci[0-9]*/**/{class,enable,irq,resource*} r,
+
+# Steam (and many games) will attempt to use lspci to learn about GPUs
+# and potentially apply workarounds on start if they detect NVIDIA.
+/{,usr}/bin/lspci ixr,
+/etc/udev/hwdb.bin r,  # Need udev data to identify GPUs
+
+/usr/share/hwdata/{pci,usb}.ids r, # Need hwdata
+
+# USB device access
+/sys/devices/pci[0-9]*/**/iad_bFunction{Sub,}Class r,
+/sys/devices/pci[0-9]*/**/bInterface{Sub,}Class r,
+/sys/devices/pci[0-9]*/**/bDevice{Sub,}Class r,
+
+# General device information access
+/sys/devices/pci[0-9]*/**/{idVendor,idProduct,manufacturer,product,serial} r,
+
+########################
+#    Feral Titles      #
+########################
+
+# Feral uses ptrace (one assumes for crash report service)
+capability sys_ptrace,
+
+# Allow confined ptrace now.
+ptrace (trace, read) peer=snap.@{SNAP_NAME}.**,
+
+# The Feral launcher script tries using curl-config, let it.
+/usr/bin/curl-config ixr,
+
+########################
+# AppArmor Limitations #
+########################
+
+# Game Dev Tycoon reads /proc/$ppid/environ but unfortunately AppArmor
+# doesn't have any such qualifier.
+@{PROC}/[0-9]*/environ r,
+`
+
+func init() {
+	registerIface(&commonInterface{
+		name:                  "steam-support",
+		summary:               steamSupportSummary,
+		implicitOnCore:        true,
+		implicitOnClassic:     true,
+		baseDeclarationSlots:  steamSupportBaseDeclarationSlots,
+		connectedPlugAppArmor: browserSupportConnectedPlugAppArmor + steamSupportConnectedPlugAppArmor,
+		connectedPlugSecComp:  browserSupportConnectedPlugSecComp + steamSupportConnectedPlugSecComp,
```

Comment:
Instead of concatenating here, can you instead change this to be (see above comments):
```
connectedPlugAppArmor: steamSupportConnectedPlugAppArmor,
connectedPlugSecComp:  steamSupportConnectedPlugSecComp,
```